opentelemetry-jaeger/src/exporter/config/agent.rs

```diff
@@ -228,25 +228,25 @@ impl AgentPipeline {
     /// Build a `TracerProvider` using a blocking exporter and configurations from the pipeline.
     ///
     /// The exporter will send each span to the agent upon the span ends.
-    pub fn build_simple(mut self) -> Result<TracerProvider, TraceError> {
-        let mut builder = sdk::trace::TracerProvider::builder();
-
-        let (config, process) = build_config_and_process(
-            builder.sdk_provided_resource(),
-            self.trace_config.take(),
-            self.transformation_config.service_name.take(),
-        );
-        let exporter = Exporter::new(
-            process.into(),
-            self.transformation_config.export_instrument_library,
-            self.build_sync_agent_uploader()?,
-        );
-
-        builder = builder.with_simple_exporter(exporter);
-        builder = builder.with_config(config);
-
-        Ok(builder.build())
-    }
+    // pub fn build_simple(mut self) -> Result<TracerProvider, TraceError> {
```

Comment:
These build simple / sync methods are the last blocker for the jaeger implementation. In order to support the non-pipelined I/O resources like the Agent uploaders, the Exporter and Uploaders were decoupled via a channel, and the uploaders now run on a dedicated task. However, this meant that `Exporter::new` has to return a task to spawn as well, similar to the stackdriver exporter. This was problematic with this `build_simple`, etc. methods since there's no runtime available on which to spawn that task.

I'll need feedback on whether we can simply thread the Runtime through here and whether this design is acceptable.

Reply:
I updated the commit to thread the runtime through, but I don't have enough context in the project to know if this is the right approach.

Reply:
Thanks for the hard work! Sorry didn't get much time yesterday to take a look. Will try to take a look tomorrow or over the weekend. We do want to keep `install_simple` without runtime parameter just so that you can work with opentelemetry without a runtime. But I need a closer look to see it's possible to do it under the new exporter APIs.

Reply:
Is it just that method in particular, or is the goal that the opentelemetry library can be used without a runtime? If it's just that method, we can probably figure something out using the cfg(feature) attrs internally.

Reply:
I think in general we want to allow people to use it without a runtime. As a util library, we shouldn't assume the runtime and deny certain use cases.